example/package.json

```diff
@@ -5,33 +5,23 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
-    "test:e2e": "vue-cli-service test:e2e"
+    "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@reststate/client": "^0.0.7",
-    "@reststate/vuex": "^0.0.16",
+    "@lucro/vuex": "^0.2.5-4",
     "axios": "^0.21.1",
```

Comment:
This is in `example` so not sure how much it matters but as of latest dependabot warning we should make the minimum `0.21.2` not `0.21.1` https://github.com/lucroglobal/loan-servicing-ui/security/dependabot/package.json/axios/open

Reply:
Updated!